cmd/reportsStorage/main.go

```diff
@@ -0,0 +1,193 @@
+package main
```

Comment:
I don't understand the reasoning well enough why we need this additional storage. Lets discuss it tomorrow in the office.

Reply:
Ok.
In short, just separation in case we store those in completely different place.

Reply:
That was the original thought... At the moment reportsStorage gives additional reports-related information on returning the file. Report file is saved with a filename built from "dataSince" and "dataUntil" timestamps and "dataSince" and "dataUntil" are returned in report file descriptor for frontend to display reports with their time coverage.